### PR TITLE
Fix for workflow merge

### PR DESCRIPTION
**Pull request by seb231**
Bug info: The further into the future the population projection for the ccm was the more likely it was that the values returned would be incorrect. Not only would the values further into the future be wrong but the earlier values (i.e. 2015) would also have changed.

This bug was eventually tracked down to a merge conflict in the workflow whereby we are currently unable to predict how two branches in the workflow are subsequently merged together. These branches have some of the same data in them, but on one branch this data has been updated. Previously there was not a predicate for the order in which these data were merged.

Subsequently when projecting further into the future, one of the branches (mortality/death prediction), which uses the parameter provided to define how far into the future to project, would take longer to run, merge second and provide the un-modified values in the population projection.

This bug has been fixed by a minor increasing the run time and reducing parallelisation, through modification to the workflow, so that the death projection is always run first.

### CODE REVIEW

File: src/witan/models/dem/ccm/models.clj [hunk 1]
```diff
@@ -91,7 +90,7 @@
      :witan/version "1.0.0"
      :witan/type :predicate
      :witan/fn :ccm-core/ccm-loop-pred
-     :witan/params {:last-proj-year 2021}}
+     :witan/params {:last-proj-year 2018}}
```
Comment (otfrom) [inline src/witan/models/dem/ccm/models.clj:93]: why the year hard coded and why the change?

Comment (seb231) [reply src/witan/models/dem/ccm/models.clj:93]: OK. So. Did you read the description above? Essentially 2018 is one of those years which is likely to return either one of the two potential values. When this catalog is used in the test it increases the chance of us having a variable result, so the test is more likely to catch it. Does that make sense?

Comment (seb231) [reply src/witan/models/dem/ccm/models.clj:93]: Also re: hard coded, all params have a default in the catalog

**Comment by otfrom** [comment]
:shipit:
